### PR TITLE
Teensy sf txt files

### PR DESCRIPTION
**Pull request by NoSloppy**
ProffieOSHumDelay set to 100ms to accomodate multiple varying length out.wavs

config.ini and smoothsw.ini updated.
- Relevant parameters added.
- Links to Wiki Pages for .ini files added.

### CODE REVIEW

File: TeensySF/ReadMe.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Teensy - a SmoothSwing V2 font by Lord Blako.
```
Comment (profezzorn) [inline TeensySF/ReadMe.txt]: git mv

File: TeensySF/config.ini [hunk 2]
```diff
@@ -1 +1,20 @@
-humstart=1890
+# TeensySF config.ini
+
+# For full descriptions of what each of these entries does,
+# along with the default values for each, see the Wiki page here:
+# https://github.com/profezzorn/ProffieOS/wiki/Sound-Font-config.ini-Configuration
+
+volHum=15
```
Comment (profezzorn) [inline TeensySF/config.ini:7]: How about we just comment out entries which have the default values?
Like: # volHum=15

Comment (NoSloppy) [reply TeensySF/config.ini:7]: I think that's a drastic change considering the millions of config.ini's already existing in the world which are formatted this way already. 
Are you sure that's how we should do this for all of these?

**Comment by NoSloppy** [comment]
Closing and resubmitting

**Pull request closed without merging**